src/components/v5/common/TableWithBurgerMenu/types.ts

```diff
@@ -0,0 +1,11 @@
+import { TableProps } from '../Table/types';
+
+export interface TableWithBurgerMenuProps
+  extends Pick<TableProps, 'tableTitle' | 'columns'> {
+  actions: {
+    fields: TableProps[];
```

Comment:
What is this used for?

Reply:
But you already have this prop in `TableProps`. Why isn't this interface used here for other fields?